src/epic.rs

```diff
@@ -302,6 +302,665 @@ pub static EPICS: &[Epic] = &[
+    },
+    Epic {
+        title: "Aurora Leigh",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Meghnad Badh Kavya",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Terje Vigen",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "La Légende des Siècles",
+        wiki: Wiki::Name("La Légende des siècles"),
+    },
+    Epic {
+        title: "The Earthly Paradise",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Ibonia",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Martín Fierro",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Idylls of the King",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Clarel",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "The Story of Sigurd the Volsung and the Fall of the Niblungs",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "L'Atlàntida",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "The Light of Asia",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "The City of Dreadful Night",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Tristram of Lyonesse",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Eros and Psyche",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "La Fin de Satan",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Canigó",
+        wiki: Wiki::Name("Canigó#Literature"),
+    },
+    Epic {
+        title: "Lāčplēsis",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Tabaré",
+        wiki: Wiki::Name("Tabaré (poem)"),
+    },
+    Epic {
+        title: "The Wanderings of Oisin",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Kotan Utunnai",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Lục Vân Tiên",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Amir Arsalan",
+        wiki: Wiki::Same,
+    },
+    // 20th century epics
+    Epic {
+        title: "The Divine Enchantment",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Lahuta e Malcís",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Ural-batyr",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "The Ballad of the White Horse",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Mensagem",
+        wiki: Wiki::Name("Fernando Pessoa#Mensagem"),
+    },
+    Epic {
+        title: "The Cantos",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Dorvyzhy",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "The Legend of Sigurd and Gudrún",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "A Cycle of the West",
+        wiki: Wiki::Name("Cycle of the West"),
+    },
+    Epic {
+        title: "The Odyssey: A Modern Sequel",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Dymer",
+        wiki: Wiki::Same,
+    },
```

Comment:
It seenms that "A" is missing (https://en.wikipedia.org/wiki/Louis_Zukofsky#.22A.22)

Note that the epic itself is titled with quotes. (So `"A"`, not `A`)